packages/tasit-action/src/TasitAction.test.js

```diff
@@ -228,7 +228,7 @@ describe("TasitAction.Contract", () => {
 
       await mineBlocks(provider, 15);
 
-      expect(fakeFn.called).to.be.true;
+      expect(fakeFn.callCount).to.be.at.most(7);
```

Comment:
Wouldn't it be exactly 7? That said, it's good to know about this `at.most` option

Reply:
Related to: https://github.com/tasitlabs/TasitSDK/pull/115#discussion_r248223253
blocks mining is faster than polling.

Reply:
Makes sense

Reply:
```suggestion
      // not exactly 7 because block mining is faster than polling
      expect(fakeFn.callCount).to.be.at.most(7);
```

Reply:
Want to quickly accept this before we merge?

Reply:
Oh never mind, I think you did